tests/phpt/fatal_error_not_captured_twice.phpt

```diff
@@ -22,7 +23,10 @@ require $vendor . '/vendor/autoload.php';
 $spool = new MemorySpool();
 $transport = new SpoolTransport($spool);
 
-init([], ClientBuilder::create()->setTransport($transport));
+init([]);
```

Comment:
I don't find very useful to use the `init` method that internally creates an instance of the client to just override it one line below. Please just use the `bindClient` method